src/main/com/capgemini/model/MenuItem.java

```diff
@@ -1,13 +1,17 @@
 package com.capgemini.model;
 
 public class MenuItem {
-    public MenuItem() {
-    }
 
     private String name;
-    private int price;
+    private double price;

```

Comment:
Check even of de valuta die hij wil in een double of integer kan. Het probleem is dat doubles en floats niet nauwkeurig zijn, dus kan het zijn dat er extra of minder centen bij het bedrag komen.

Reply:
Dit ga ik morgen vragen dan.